akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -44,6 +44,8 @@ object ClusterSingletonManagerSettings {
    */
   def apply(system: ActorSystem): ClusterSingletonManagerSettings =
     apply(system.settings.config.getConfig("akka.cluster.singleton"))
+      // note that this setting has some additional logic inside the ClusterSingletonManager
+      // falling back to DowningProvider.downRemovalMargin if it is off/Zero
```

Comment:
Does that result in an precedence of akka.cluster.down-removal-margin -> downingProvider.downRemovalMargin ?

Reply:
that is the purpose, use settings.DownRemovalMargin if that is defined, otherwise use DowningProvider.downRemovalMargin